change-notes/1.20/analysis-cpp.md

```diff
@@ -0,0 +1,16 @@
+# Improvements to C/C++ analysis
+
+## General improvements
+
+## New queries
+
+| **Query**                   | **Tags**  | **Purpose**                                                        |
+|-----------------------------|-----------|--------------------------------------------------------------------|
+
+## Changes to existing queries
+
+| **Query**                  | **Expected impact**    | **Change**                                                       |
+|----------------------------|------------------------|------------------------------------------------------------------|
+| Unused static variable | Fewer false positive results | Variables with the attribute `unused` are now excluded from the query. |
```

Comment:
I think we should add query ids here as soon as we add entries. Then @felicity-semmle won't have to do detective work to reconstruct them just before the release.

Reply:
I agree.  Done.